modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala

```diff
@@ -6,276 +6,720 @@ package observe.common
         observations(programId: "p-2") {
           nodes {
             id
-            observationTarget {
-              ... on Target {
-                targetId: id
-                targetName: name
-              }
-              ... on Asterism {
-                asterismId: id
-                asterismName: name
-              }
-            }
-            constraintSet {
-              imageQuality
-              cloudExtinction
-              skyBackground
-              waterVapor
-            }
-            status
-            activeStatus
-            plannedTime {
-              execution {
-                microseconds
-              }
-            }
+            name
           }
         }
-
-        constraintSetGroup(programId: "p-2") {
-            nodes {
-              constraintSet {
-                cloudExtinction
-                imageQuality
-                skyBackground
-                waterVapor
-                elevationRange {
-                  type: __typename
-                  ... on AirMassRange {
-                    min
-                    max
-                  }
-                  ... on HourAngleRange {
-                    minHours
-                    maxHours
-                  }
-                }
-              }
-              observations {
-                nodes {
-                  id
-                }
-              }
-            }
-          }
-        }
-    """
-
-    object Data {
-      object Observations {
-        object Nodes {
-          object PlannedTime {
-            type Execution = time.Duration
-          }
-        }
-      }
-    }
-
-  }
-
-  @GraphQL
-  trait ProgramObservationsEditSubscription extends GraphQLOperation[ObservationDB] {
-    val document = """
-      subscription {
-        observationEdit(programId:"p-2") {
-          id
-        }
-      }
-    """
-  }
-
-  @GraphQL
-  trait ProgramCreateObservation extends GraphQLOperation[ObservationDB] {
-    val document = """
-      mutation($createObservation: CreateObservationInput!) {
-        createObservation(input: $createObservation) {
-          id
-        }
       }
     """
   }
 
   @GraphQL
-  trait ProgramDeleteObservation extends GraphQLOperation[ObservationDB] {
-    val document = """
-      mutation($oid: ObservationId!) {
-        deleteObservation(observationId: $oid) {
-          id
-        }
-      }
-    """
-  }
-
-  @GraphQL
-  trait ProgramUndeleteObservation extends GraphQLOperation[ObservationDB] {
-    val document = """
-      mutation($oid: ObservationId!) {
-        undeleteObservation(observationId: $oid) {
-          id
-        }
-      }
-    """
-  }
-
-  @GraphQL
-  trait ObsEditQuery extends GraphQLOperation[ObservationDB] {
+  trait ObsQuery extends GraphQLOperation[ObservationDB] {
     val document = """
       query($obsId: ObservationId!) {
         observation(observationId: $obsId) {
           id
+          name
+          targets {
+            firstScienceTarget {
```

Comment:
Why do we fetch only the first target?

Reply:
Mainly because I don't understand the data model for targets. But Seqexec needs it to verify that the TCS was configured with the right one. On second thought, that may be not necessary in GPP if Observe and Engage talk to each other to prevent this kind of error.

Reply:
An observation may have multiple `scienceTarget`s, which is how we are modelling asterisms.